src_py/_debug.py

```
@@ -136,51 +176,32 @@ def default_return(linked=True):
-        debug_str += f"Mixer Driver:\t\t{get_mixer_driver()}"
-    else:
-        debug_str += "Mixer Driver:\t\tMixer Not Initialized"
+    Args:
+        filename [DEPRECATED]: string name of the file to save
+    """
+    import warnings
 
-    if filename is None:
-        print(debug_str)
+    if filename is not None:
+        warnings.warn("filename parameter is deprecated, printing to console.")
```

Ruff (0.16.2)

[warning] No explicit `stacklevel` keyword argument found

Set `stacklevel=2`

(B028)